<?xml version="1.0"?>
<launch>

<node name="mesh_publisher" pkg="lvr_ros_bridge" type="mesh_publisher" output="screen">

	<param name="mesh_topic" value="/triangle_mesh"/>
	<param name="mesh_frame" value="/mesh_link"/>
	<param name="to_file_topic" value="~/mesh1.ply"/>
	<param name="to_file_path" value="~/"/>
	

</node>
</launch>



